src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,29 @@
+package duke;
+
+import duke.exception.*;
```

Comment:
Avoid wildcard imports (.*). You can configure the setting for imports in the IDE.